[2018-08-15 11:26:03,876] Spencers-MacBook-Pro.local/ERROR/stderr: KeyboardInterrupt
[2018-08-15 11:26:03,885] Spencers-MacBook-Pro.local/ERROR/stderr: 2018-08-15T17:26:03Z
[2018-08-15 11:26:03,885] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:26:33,995] Spencers-MacBook-Pro.local/ERROR/stderr: KeyboardInterrupt
[2018-08-15 11:26:33,995] Spencers-MacBook-Pro.local/ERROR/stderr: 2018-08-15T17:26:33Z
[2018-08-15 11:26:33,995] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:27:31,500] Spencers-MacBook-Pro.local/ERROR/stderr: KeyboardInterrupt
[2018-08-15 11:27:31,500] Spencers-MacBook-Pro.local/ERROR/stderr: 2018-08-15T17:27:31Z
[2018-08-15 11:27:31,500] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:17,694] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:30:17,694] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:30:17,695] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:17,695] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:30:17,695] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:30:17,695] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:17,695] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:30:17,695] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:30:17,696] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:30:17,697] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:30:17,697] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:30:17,697] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:30:17,697] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:30:17,697] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:30:17,697] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:25,407] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:30:25,408] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:32:49,236] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:32:49,237] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:32:56,836] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:32:56,837] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:32:56,838] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:32:56,838] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:39,535] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:33:39,536] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:33:39,536] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:39,537] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:33:39,537] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:33:39,537] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:39,537] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:33:39,539] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:33:39,540] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:39,540] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:33:39,540] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:33:39,541] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:39,541] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:33:39,541] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:33:39,542] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:33:39,542] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:33:39,542] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:33:39,542] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:33:39,543] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:33:39,543] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:33:39,543] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:48,174] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:33:48,174] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:33:48,174] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:48,174] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:33:48,174] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:33:48,174] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:33:48,175] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:33:48,176] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:00,401] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:34:00,402] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:34:53,971] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:34:53,972] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:34:53,973] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:34:53,973] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:34:53,973] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:34:53,973] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:34:53,973] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:08,053] Spencers-MacBook-Pro.local/ERROR/stderr: Traceback (most recent call last):
[2018-08-15 11:35:08,053] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/bin/locust", line 11, in <module>
[2018-08-15 11:35:08,053] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:08,053] Spencers-MacBook-Pro.local/ERROR/stderr: load_entry_point('locustio==0.9.0.dev0', 'console_scripts', 'locust')()
[2018-08-15 11:35:08,053] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/main.py", line 463, in main
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: runners.locust_runner = MasterLocustRunner(locust_classes, options)
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/runners.py", line 248, in __init__
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: self.server = rpc.Server(self.master_bind_host, self.master_bind_port)
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: File "/Users/spencerpinegar/PycharmProjects/Performance_Test/venv/lib/python2.7/site-packages/locust/rpc/zmqrpc.py", line 20, in __init__
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: self.receiver.bind("tcp://%s:%i" % (host, port))
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/socket.pyx", line 549, in zmq.backend.cython.socket.Socket.bind
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: zmq.error
[2018-08-15 11:35:08,054] Spencers-MacBook-Pro.local/ERROR/stderr: .
[2018-08-15 11:35:08,055] Spencers-MacBook-Pro.local/ERROR/stderr: ZMQError
[2018-08-15 11:35:08,055] Spencers-MacBook-Pro.local/ERROR/stderr: :
[2018-08-15 11:35:08,055] Spencers-MacBook-Pro.local/ERROR/stderr: Address already in use
[2018-08-15 11:35:08,055] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:12,890] Spencers-MacBook-Pro.local/ERROR/stderr: KeyboardInterrupt
[2018-08-15 11:35:12,890] Spencers-MacBook-Pro.local/ERROR/stderr: 2018-08-15T17:35:12Z
[2018-08-15 11:35:12,890] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:35:57,362] Spencers-MacBook-Pro.local/ERROR/stderr: KeyboardInterrupt
[2018-08-15 11:35:57,362] Spencers-MacBook-Pro.local/ERROR/stderr: 2018-08-15T17:35:57Z
[2018-08-15 11:35:57,362] Spencers-MacBook-Pro.local/ERROR/stderr: 
[2018-08-15 11:39:30,572] Spencers-MacBook-Pro.local/ERROR/stderr: KeyboardInterrupt
[2018-08-15 11:39:30,572] Spencers-MacBook-Pro.local/ERROR/stderr: 2018-08-15T17:39:30Z
[2018-08-15 11:39:30,572] Spencers-MacBook-Pro.local/ERROR/stderr: 
